docs/installation.html

```diff
@@ -0,0 +1,82 @@
+<!DOCTYPE html>
+<html>
+<head>
+<link rel="stylesheet" type="text/css" href="cbat.css">
+<title>CBAT Installation</title>
+</head>
+<body>
+
+<h1>CBAT Installation</h1>
+
+<section>
+
+<p>Table of contents:
+
+<ul>
+  <li><a href="#docker">Docker Installation</a></li>
+  <li><a href="#manual">Manual Installation</a></li>
+</ul>
+</section>
+
+<section>
+
+<a name="docker"></a>
+<h2>Docker Installation</h2>
+
+<p>Coming soon.
+
+</section>
+
+<section>
+
+<a name="manual"></a>
+<h2>Manual Installation</h2>
+
+<h3>Prerequisites</h3>
+
+<p>Before installing, make sure you have the appropriate dependencies installed. First, there is OCaml:
+
+<ul>
+    <li>ocaml 4.07.0</li>
+</ul>
+
+<p>Then, there are the following OCaml packages:
+
+<ul>
+    <li>core_kernel 0.12.4+</li>
+    <li>bap 2.1.0</li>
+    <li>ounit 2.0.8</li>
+    <li>z3 4.8.8</li>
+    <li>re 1.9.0</li>
+</ul>
+
+<p>All of these can be installed with:
+
+<pre>
+  opam install package_name
```

Comment:
We should really have a command or sequence of commands that you can just copy/paste to install all the prereqs.